src/classes/User.js

```diff
@@ -0,0 +1,41 @@
+class User {
+  constructor(userData = {}) {
+    this.id = userData.id;
+    this.name = userData.name;
+    this.pantry = userData.pantry;
```

Comment:
Could you give this a default value to fall back to? I imagine it would be an empty array i.e. []

Reply:
Are you thinking a || [ ] for the pantry?

Reply:
We got this done, and added a test for a default pantry of  [ ].